pubspec.yaml

```diff
@@ -1,46 +1,46 @@
 name: pana
 description: Evaluate the health and quality of a Dart package
-version: 0.17.0
+version: 0.17.1-dev
 homepage: https://github.com/dart-lang/pana
 
 environment:
-  sdk: '>=2.10.0 <3.0.0'
+  sdk: '>=2.11.99 <3.0.0'
 
 dependencies:
-  analyzer: '>=0.41.0 <2.0.0'
-  args: '>=1.6.0 <3.0.0'
-  async: '>=1.13.3 <3.0.0'
-  cli_util: '>=0.2.0 <0.4.0'
-  html: '>=0.13.3 <0.16.0'
-  http: '>=0.11.3 <0.14.0'
-  io: '>=0.3.4 <2.0.0'
-  json_annotation: '>=2.0.0 <5.0.0'
-  logging: '>=0.11.3+1 <2.0.0'
-  markdown: '>=3.0.0 <5.0.0'
+  analyzer: ^1.0.0
```

Comment:
Please don't force-update these dependencies yet, as it will be incompatible with `pub-dev`, and we'll be force-upgrade with too many breaking changes.

Reply:
Do we have a choice?